### PR TITLE
HOT FIX FOR EXPORTS

### PR DESCRIPTION
**Pull request by deadlybutter**
#### What's this PR do?
Get user data in bulk from repository
Write users in bulk to csv
Add no RB data option

#### How should this be manually tested?
does it export the things right

#### Any background context you want to provide?
see slacks

#### What are the relevant tickets?
Fixes # 🍰

### CODE REVIEW

**Comment by weerd** [comment]
```
$csv = $this->manager->exportCSV($competition, true);
```
Isn't this the line that needs to get changed to be false?

**Comment by deadlybutter** [comment]
@weerd sorry realized what you meant, updated now :D

**Comment by deadlybutter** [comment]
and export without reportback data is like super fast in comparison to now

File: app/Http/Controllers/CompetitionsController.php [hunk 1]
```diff
@@ -104,9 +105,9 @@ public function destroy(Competition $competition)
      * @param  \Gladiator\Models\Competition  $competition
      * @return \League\Csv\ $csv
      */
-    public function export(Competition $competition)
+    public function export(Competition $competition, Request $request)
     {
-        $csv = $this->manager->exportCSV($competition, true);
+        $csv = $this->manager->exportCSV($competition, $request->input('hasReportback'));
```
Comment (weerd) [inline app/Http/Controllers/CompetitionsController.php]: maybe use a ternary to set as `false` if nothing there instead of `null`?

Comment (deadlybutter) [inline app/Http/Controllers/CompetitionsController.php]: we can if you think it looks better, but functionally it doesnt make a difference

File: app/helpers.php [hunk 2]
```diff
@@ -11,10 +11,7 @@
 function build_csv($data)
 {
     $csv = \League\Csv\Writer::createFromFileObject(new \SplTempFileObject());
-
-    foreach ($data as $row) {
-        $csv->insertOne($row);
-    }
+    $csv->insertAll($data);
```
Comment (weerd) [inline app/helpers.php]: mind adding a space above :) nit-pick :diego:

File: app/Http/Controllers/CompetitionsController.php (hunk 1 above)
Comment (weerd) [inline app/Http/Controllers/CompetitionsController.php]: actually you're specifically passing it in in both cases. Guess it's fine for now.

**Comment by deadlybutter** [comment]
added line above, i think thats what you meant

**Comment by deadlybutter** [comment]
that addition caused style CI to fail?

**Comment by weerd** [comment]
@deadlybutter it's not the additional line... it's the empty tab in the space that caused it to fail.

**Comment by deadlybutter** [comment]
@deadlybutter install editorconfig note to self

@weerd fixed!

**Comment by weerd** [comment]
👍

**Pull request merged**